### PR TITLE
feat: upload files to storage-api

### PR DESCRIPTION
**Pull request by MaxwelMazur**
- upload files on storageapi

### CODE REVIEW

**Review by ftoldato — APPROVED** [review]
nice Job

File: pkg/cmd/edge_applications/publish/publish.go [hunk 1]
```diff
@@ -1,600 +1,652 @@
...
+        return nil
+    }); err != nil {
+        return err
+    }
+
+    clientUpload := storage.NewClient(f.HttpClient, f.Config.GetString("storage_url"), f.Config.GetString("token"))
+
+    currentFile := 0
+    if err = cmd.FilepathWalk(pathStatic, func(path string, info os.FileInfo, err error) error {
+        defer time.Sleep(time.Millisecond*1)
```
Comment (PatrickMenoti) [inline pkg/cmd/edge_applications/publish/publish.go:142]: why the sleep here?

File: pkg/cmd/edge_applications/publish/publish.go [hunk 2]
```diff
@@ -1,600 +1,652 @@
...
+            return err
+        }
+        if !info.IsDir() {
+            fileContent, err := cmd.Open(path)
+            if err != nil {
+                return err
+            }
+
+            fileString := strings.TrimPrefix(path, pathStatic)
+            if err = clientUpload.Upload(context.Background(), versionID.String(), fileString, fileString, fileContent); err != nil {
```
Comment (PatrickMenoti) [inline pkg/cmd/edge_applications/publish/publish.go:153]: why is fileString sent twice?

**Review by PatrickMenoti — COMMENTED** [review]
Also, we need to put Storage-API url in prod env file

File: pkg/cmd/edge_applications/publish/publish.go (hunk 2 above)
Comment (MaxwelMazur) [reply pkg/cmd/edge_applications/publish/publish.go]: me fail!

**Review by PatrickMenoti — APPROVED** [review]

**Pull request merged**